ex4_ch1.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+noun = input("Enter a noun. ")
+
+verb = input("Enter a verb. ")
+
+adverb = input("Enter an adverb. ")
+
+adjective = input("Enter an adjective. ")
+
+print(f"Your {adjective} {noun} {verb} {adverb}. That's hilarious!")
```

Comment:
It's not a big deal as you get the gist of interpolating the variables in the `print` function, but it doesn't look like the string matches what the solution should be:

```
Do you walk your blue dog quickly? That's hilarious!
```